Userland/Libraries/LibVideo/VP9/MotionVector.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
The MotionVector change could be split into a separate commit.

Reply:
I've split this commit up into:

1. Rename MV->MotionVector
2. Make MotionVector.h allow constexpr and create MotionVector tables
3. The rest of the inter frame parsing commit
4. Some bug fixes on old code that were included in the big inter frame commit previously